modules/light-clients/07-tendermint/types/store.go

```diff
@@ -131,6 +134,43 @@ func deleteProcessedTime(clientStore sdk.KVStore, height exported.Height) {
 	clientStore.Delete(key)
 }
 
+// ProcessedHeight Store code
```

Comment:
My preference would be to remove this comment and same comment for processed time, the function names in `store.go` seem self descriptive to me